src/openzaak/components/zaken/admin/zaken.py

```diff
@@ -346,18 +365,29 @@ def clean(self):
 
 @admin.register(RelevanteZaakRelatie)
 class RelevanteZaakRelatieAdmin(admin.ModelAdmin):
-    list_display = ("zaak", "_relevant_zaak", "_relevant_zaak_url", "aard_relatie")
+    list_display = (
+        "zaak",
+        "_relevant_zaak",
+        "_relevant_zaak_base_url",
+        "_relevant_zaak_relative_url",
+        "aard_relatie",
+    )
     list_filter = ("aard_relatie",)
     search_fields = (
         "zaak__uuid",
         "zaak__identificatie",
         "_relevant_zaak__uuid",
         "_relevant_zaak__identificatie",
-        "_relevant_zaak_url",
+        "_relevant_zaak_relative_url",
     )
     form = RelevanteZaakRelatieForm
-    ordering = ("zaak", "_relevant_zaak", "_relevant_zaak_url")
-    raw_id_fields = ("zaak", "_relevant_zaak")
+    ordering = (
+        "zaak",
+        "_relevant_zaak",
+        "_relevant_zaak_base_url",
+        "_relevant_zaak_relative_url",
+    )
+    raw_id_fields = ("zaak", "_relevant_zaak", "_relevant_zaak_base_url")
     list_select_related = ("zaak", "_relevant_zaak")
```

Comment:
please add the `_relevant_zaak_base_url` FK here too, otherwise performance will suffer